package ru.geekbrains.geekkotlin.data.errors

class NoAuthException : Throwable()